packages/mobile/src/utils/formatting.ts

```diff
@@ -32,16 +32,47 @@ export const getFeeDisplayValue = (value: BigNumber.Value | null | undefined): s
   return value ? numeral(BigNumber.max(value, 0.001).toNumber()).format('0[.][0000]') : ''
 }
 
+/**
+ * More precise getFeeDisplayValue with built in rounding
+ * Used for small Network Fees in transaction feed
+ * @param value fee amount
+ */
+export const getNetworkFeeDisplayValue = (value: BigNumber.Value): string => {
+  const roundedNumber = new BigNumber(value)
+  if (roundedNumber.isLessThan(0.001)) {
+    return '<0.001'
+  } else {
+    return numeral(roundUp(value, 3).toNumber()).format('0[.][000]')
+  }
+}
+
+/**
+ * Even more precise getFeeDisplay value for Network Fee drilldown
+ * Built in rounding below <0.000001. Displays fee to 6 decimal
+ * places if less than 0.001, else to 3 decimal places
+ * @param value fee amount
+ */
+export const getPreciseNetworkFeeDisplayValue = (value: BigNumber.Value): string => {
```

Comment:
Similar feedback here. I've gone through this file on 2 different occasions to reduce down the number of formatting utils because people tend to just add new ones for each new case. I understand the reasoning but it makes it harder to enforce consistency of number presentation as the number of formatters grows.